commands/import_installation_test.go

```diff
@@ -192,7 +194,7 @@ var _ = Describe("ImportInstallation", func() {
 			Expect(fmt.Sprintf(format, v...)).To(Equal("finished import"))
 		})
 
-		It("it only retries 3 times before giving up", func(done Done) {
+		It("it only retries 3 times before giving up", func(ctx context.Context) {
```

Comment:
Is the Context in the test function necessary? It doesn't seem to be used.

Reply:
that's the way to test async behavior in ginkgo, without the context being passed in the callback, we get this error 
```
Invalid NodeTimeout SpecTimeout, or GracePeriod
It("it only retries 3 times before giving up", func() {
/Users/hyayi/work/om/commands/import_installation_test.go:197
  [It] was passed NodeTimeout, SpecTimeout, or GracePeriod but does not have a
  callback that accepts a SpecContext or context.Context.  You must accept a
  context to enable timeouts and grace periods

  Learn more at:
  http://onsi.github.io/ginkgo/#spec-timeouts-and-interruptible-nodes

```

Reply:
I was expecting to see the Context being used to actually terminate the test when it times out (as in the example in the docs). It seems like this test would ignore the time out and continue to run until completion.

Reply:
NodeTimeout/SpecTimeout terminates the spec, I think that should be enough for this test. However, I refactored this test to use context with a timeout, which should also timeout the process the test triggered. Let me know what you think